crates/rover-client/src/error.rs

```diff
@@ -163,6 +163,17 @@ pub enum RoverClientError {
 
     #[error("This endpoint doesn't support subgraph introspection via the Query._service field")]
     SubgraphIntrospectionNotAvailable,
+
+    #[error("The input provided is invalid")]
```

Comment:
This is an interesting error! I assume the back end provides a bit more context here - could you provide an example of when this error would occur and what the full Rover output would be?

Reply:
++ still need this

Reply:
The sources of this error are "Graph $graphRef has no published schema. Please publish a schema first" and "Graph $graphRef is not a composition variant."
I've added a suggestion in this scenario telling users: "Graph {graphref} has no published schema or is not a composition variant. Please publish a schema or use a different variant."